common/build.gradle

```diff
@@ -169,6 +169,13 @@ dependencies {
     testImplementation 'org.json:json:20160810'
     testImplementation "junit:junit:$rootProject.ext.junitVersion"
     testImplementation "org.mockito:mockito-core:$rootProject.ext.mockitoCoreVersion"
+    testImplementation "io.mockk:mockk:1.11.0"
```

Comment:
Let's move the version numbers of `io.mockk.mockk`,`io.mockk:mockk-agent-jvm` and `kotlin-reflect` to `gradle/versions.gradle`

Reply:
This will have to wait until we modify the `gradle/versions.gradle` file in  Android complete, ADAL and Broker repositories